.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+  - 3.6
+services:
+  - docker
+install:
+  - docker build -t max-text-summarizer .
+  - docker run -it -d --rm -p 5000:5000 max-text-summarizer
+  - pip install -r requirements-test.txt
+before_script:
+  - sleep 30
+script:
+  - flake8 . --max-line-length=127 --exclude=./core/getpoint
```

Comment:
Can you move this to right before sleep, that way if the build fails due to a minor style issue we find out sooner?